tests/Functional/Doctrine/Orm/RedirectRouteTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Bundle\RoutingBundle\Tests\Functional\Doctrine\Orm;
+
+use Symfony\Cmf\Bundle\RoutingBundle\Doctrine\Orm\Route;
+use Symfony\Cmf\Bundle\RoutingBundle\Doctrine\Orm\RedirectRoute;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\RouteCollection;
+
+class RedirectRouteTest extends OrmTestCase
+{
+    private $repository;
+
+    public function setUp()
+    {
+        parent::setUp();
+        $this->clearDb(Route::class);
```

Comment:
No idea why, but should this be a `RedirectRoute`?

Reply:
will add clearing for RedirectRoute as well but don't think this is the main issue...